portfolio/src/main/java/com/google/sps/servlets/data/PersonalDetails.java

```diff
@@ -28,7 +27,9 @@ public int getCurrentAge() {
     return currentAge;
   }
 
-  public String getSchoolYear() {
-    return schoolYear;
+  public String getRandomFact() {
+    int idx = new Random().nextInt(randomFacts.length);
```

Comment:
in this case of getting random index, Math.random() is used more often (see notes in https://docs.oracle.com/javase/8/docs/api/java/util/Random.html)